osu-winello.sh

```diff
@@ -442,6 +444,20 @@ Icon=$HOME/.local/share/icons/osu-wine.png" | tee "$HOME/.local/share/applicatio
 #   =====================================
 #   =====================================
 
+# just a function to not restart the game after installing
+function Terminal(){
```

Comment:
[this](https://github.com/NelloKudo/osu-winello/blob/e9684e7aab04a11664a2c3bbfe225b8a14412ffd/osu-winello.sh#L92) exists already, try to avoid adding duplicated code